proto/cronos/cronos.proto

```diff
@@ -7,6 +7,18 @@ option go_package = "github.com/crypto-org-chain/cronos/x/cronos/types";
 
 // Params defines the parameters for the cronos module.
 message Params {
-    option (gogoproto.goproto_stringer)             = false;
-    string                  ibc_cro_denom           = 1 [(gogoproto.moretags) = "yaml:\"ibc_cro_denom,omitempty\""];
+  option (gogoproto.goproto_stringer) = false;
+  string ibc_cro_denom = 1
+      [ (gogoproto.moretags) = "yaml:\"ibc_cro_denom,omitempty\"" ];
+}
+
+// TokenMappingChangeProposal defines a proposal to change one token mapping.
+message TokenMappingChangeProposal {
+  option (gogoproto.goproto_getters) = false;
+  option (gogoproto.goproto_stringer) = false;
+
+  string title = 1;
+  string description = 2;
+  string denom = 3;
```

Comment:
I wonder if it would be not better to support multiple token mapping change so that we don't need to approve x times for a batch of x token supported?

Reply:
Hmm, we could also create multiple proposals at once, and do the voting parallelly?

Reply:
yeah but voting parallely might be an hassle if they are 10+ proposal at the same time

but anyways, its not a big issue. We can change it later if it is requested.